### DISCUSSION TITLE
Trick mode watch de node pour exécuter le code d'exos

### DISCUSSION THREAD

**Discussion by samuelroland** [web-classroom/web-classroom.github.io#55] [Q&A : Théorie / Cours]
Je tenais à vous partager un moyen intéressant d'exécuter du code pour les petits exos faits en classe (sumOfSquares() par exemple), sans devoir constamment relancer l'exécution et sans utiliser la console du navigateur (qui malgré l'aperçu en temps réel, ne permet facilement d'éditer plusieurs lignes autant bien qu'un IDE).

C'est le flag expérimental de node `--watch` qui relance l'exécution du script dès que le fichier est sauvé. En voici un exemple:
![image](https://github.com/web-classroom/web-classroom.github.io/assets/47849646/c68e6f3a-d9de-4b1f-89cc-beb0a6e3f22c)

Cela ne fonctionne qu'avec les programmes qui se terminent (cela ne fonctionne pas les serveurs en labo qui écoutent sur un port...)

**Comment by Nortalle** [comment]
Thanks !
